src/platform/silabs/efr32/BUILD.gn

```diff
@@ -77,10 +78,20 @@ static_library("efr32") {
     sources += [ "BLEManagerImpl.cpp" ]
   }
 
-  if (chip_enable_ota_requestor) {
+  if (chip_enable_multi_ota_requestor) {
+    sources += [
+      "${silabs_platform_dir}/multi-ota/OTAMultiImageProcessorImpl.cpp",
+      "${silabs_platform_dir}/multi-ota/OTAMultiImageProcessorImpl.h",
+      "${silabs_platform_dir}/multi-ota/OTATlvProcessor.cpp",
+      "${silabs_platform_dir}/multi-ota/OTATlvProcessor.h",
+      "${silabs_platform_dir}/multi-ota/efr32/OTAFirmwareProcessor.cpp",
+      "${silabs_platform_dir}/multi-ota/efr32/OTAFirmwareProcessor.h",
+      "${silabs_platform_dir}/multi-ota/efr32/OTAHooks.cpp",
+    ]
+  } else if (chip_enable_ota_requestor) {
     sources += [
       "${silabs_platform_dir}/OTAImageProcessorImpl.h",
-      "OTAImageProcessorImpl.cpp",
+      "${silabs_platform_dir}/efr32/OTAImageProcessorImpl.cpp",
```

Comment:
I guess it leads to the same result but `OTAImageProcessorImpl.cpp` is in the same folder as this build.gn which is `${silabs_platform_dir}/efr32/`. You can leave it with no additional path here.